fiftyone/server/routes/tagging.py

```
@@ -53,37 +62,258 @@ async def post(self, request: Request, data: dict) -> dict:
+                ]
+            )
+
+        all_tags = set()
+        for doc in results:
+            for key, value in doc.items():
+                for entry in value:
+                    if entry["_id"] is not None:
+                        all_tags.add(entry["_id"])
+
+        return sorted(all_tags)
+    else:
```

Ruff (0.15.9)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)